2024-03-31 16:36:22,639 | INFO | requests_utils.py: 7|发送请求的url为：http://systest.whsaviour.com/platform-admin/sys/login
2024-03-31 16:36:22,639 | INFO | requests_utils.py: 8|发送请求的method为：post
2024-03-31 16:36:22,639 | INFO | requests_utils.py: 9|发送请求的headers为：{"Content-Type":"application/json"}
2024-03-31 16:36:22,640 | INFO | requests_utils.py: 10|发送请求的data为：{'username': 'admin', 'password': '10010.com'}
2024-03-31 16:36:22,868 | INFO | requests_utils.py: 24|响应状态码为：200
2024-03-31 16:36:22,869 | INFO | requests_utils.py: 25|响应success为：True
2024-03-31 16:36:22,869 | INFO | requests_utils.py: 26|响应message为：登录成功
2024-03-31 16:36:22,874 | INFO | requests_utils.py: 7|发送请求的url为：http://systest.whsaviour.com/platform-admin/sys/login
2024-03-31 16:36:22,874 | INFO | requests_utils.py: 8|发送请求的method为：post
2024-03-31 16:36:22,874 | INFO | requests_utils.py: 9|发送请求的headers为：{"Content-Type":"application/json"}
2024-03-31 16:36:22,874 | INFO | requests_utils.py: 10|发送请求的data为：{'username': 'admin', 'password': '10010'}
2024-03-31 16:36:22,918 | INFO | requests_utils.py: 24|响应状态码为：200
2024-03-31 16:36:22,918 | INFO | requests_utils.py: 25|响应success为：False
2024-03-31 16:36:22,918 | INFO | requests_utils.py: 26|响应message为：用户名或密码错误
2024-03-31 16:36:22,922 | INFO | requests_utils.py: 7|发送请求的url为：http://systest.whsaviour.com/platform-admin/sys/login
2024-03-31 16:36:22,922 | INFO | requests_utils.py: 8|发送请求的method为：post
2024-03-31 16:36:22,922 | INFO | requests_utils.py: 9|发送请求的headers为：{"Content-Type":"application/json"}
2024-03-31 16:36:22,922 | INFO | requests_utils.py: 10|发送请求的data为：{'username': 'admin', 'password': '1'}
2024-03-31 16:36:22,966 | INFO | requests_utils.py: 24|响应状态码为：200
2024-03-31 16:36:22,967 | INFO | requests_utils.py: 25|响应success为：False
2024-03-31 16:36:22,967 | INFO | requests_utils.py: 26|响应message为：用户名或密码错误
